cli/install/template.go

```diff
@@ -415,6 +423,8 @@ spec:
             cpu: 300m
             memory: 300Mi
         {{- end }}
+        securityContext:
+          runAsUser: 12102
```

Comment:
When I try to install the prometheus pod with this user ID set, the pod ends up in CrashLoopBackoff, with the following error:

```
level=error ts=2018-12-06T00:57:56.2975388Z caller=main.go:617 err="opening storage failed: mkdir data/: permission denied"
```

Based on my reading of prometheus/prometheus#3441, it looks like the pod is already configured to run as a non-root user:

```
$ docker run --rm --entrypoint id prom/prometheus:v2.4.0
uid=65534(nobody) gid=65534(nogroup)
```

And setting that user ID here seems to fix the issue:

```
        securityContext:
          runAsUser: 65534
```

Reply:
So here, I will just hard-code this value in the template.